packages/tools/src/tools/OrientationMarkerTool.ts

```diff
@@ -94,6 +94,38 @@ class OrientationMarkerTool extends BaseTool {
     this.initViewports();
   };
 
+  onSetToolDisabled = (): void => {
+    this.cleanUpData();
+  };
+
+  private cleanUpData() {
+    const renderingEngines = getRenderingEngines();
+    const renderingEngine = renderingEngines[0];
+    const viewports = renderingEngine.getViewports();
+
+    viewports.forEach((viewport) => {
+      const orientationMarker = this.orientationMarkers[viewport.id];
+      if (!orientationMarker) {
+        return;
+      }
+
+      const { actor, orientationWidget } = orientationMarker;
+      orientationWidget?.setEnabled(false);
+      orientationWidget?.delete();
+      actor?.delete();
+
+      const renderer = viewport.getRenderer();
+      const renderWindow = viewport
+        .getRenderingEngine()
+        .offscreenMultiRenderWindow.getRenderWindow();
+      renderer.resetCamera();
```

Comment:
why we are reseting the viewport camera here?

Reply:
No particular reason other than that I was just doing the same thing as what happens on the `addAxisActorInViewport` function below.

But I can remove that if it's not necessary.

Reply:
Yes that will cause unexpected behaviour

Reply:
@sedghi I've removed it now.

Reply:
Is the `resetCamera()` call necessary in `addAxisActorInViewport`? That's the thing causing unexpected behavior for me. Not sure the original intent of having it there and wouldn't want other expected behavior to break but if it's not needed can we remove it as well?